ivy-scaling-nginx/nginx.conf

```diff
@@ -0,0 +1,44 @@
+events {
+  worker_connections  1024;
+}
+
+http {
+  upstream ivy-cluster {
+    ip_hash;
+    server 192.168.80.4:8080;
```

Comment:
hard coded ips. does this work also on my system?

Reply:
instead use ivy:8080

Reply:
Are you sure this works with sticky sessions? I don't think the ip_hash will be used when just configuring ivy:8080.

The IPS should work on any system as I configure the subnet in docker-compose.yaml.

Reply:
ip_hash uses the client ip address to use sticky sessions not the server ip address. Not sure if ivy:8080 works correctly with NGNIX. Normally ivy as host should work because in a docker network the host names are as the service names. If more than one container is started for a service both ip addresses are available when resolving the name ivy. To be sure you have to test it.

Reply:
Seems to work with ivy:8080, I got into different node, not just the first one, and got redirected there.